BuildMonitor/BuildMonitor.csproj

```diff
@@ -62,15 +63,22 @@
     <Compile Include="Domain\SolutionBuild.cs" />
     <Compile Include="Domain\SolutionBuildData.cs" />
     <Compile Include="IntExtensions.cs" />
+    <Compile Include="LocalData\AnalyseBuildTimes.cs" />
+    <Compile Include="LocalData\BuildTimes.cs" />
     <Compile Include="LocalData\DataAdjuster.cs" />
+    <Compile Include="LocalData\IBuildTimes.cs" />
     <Compile Include="LocalData\InvalidData.cs" />
+    <Compile Include="LocalData\JSONSolutionName.cs" />
+    <Compile Include="LocalData\JSONSolutionTimes.cs" />
     <Compile Include="LocalData\RawBuildData.cs" />
+    <Compile Include="LocalData\SolutionMonth.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="UI\LongExtensions.cs" />
     <Compile Include="UI\Time.cs" />
     <Compile Include="UI\TimePresenter.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="cedd.snk" />
     <None Include="key.pfx" />
```

Comment:
It depends. The snk is used for strong-naming, which uses a key pair to uniquely identify an assembly. The pfx is for code signing, which is a similar process but one that is intended to prevent malicious tampering with assemblies that are distributed publicly. Using both on an assembly both strong-names AND signs it.

It would probably be best to compile and release it using your original pfx, but I can't do this as I don't have the password.